README.md

```diff
@@ -11,19 +11,22 @@ For more information, find the full documentation [here](https://docs.getliteral
 
 ### TypeScript
-| Name | Category | Description| 
-| --- | --- | --- |
-| [Prompt Iteration with Promptfoo](/typescript/prompt-iteration-promptfoo/) | Evaluation | Run a simple chat application and evaluate its results with two different prompt templates with Promptfoo. |
-| [Image Generation with Dall-E & Literal AI Attachment](/typescript/dall-e-image-generation) | Observablity | Use OpenAI Dall-E to Generate an image with a prompt and then send it to Literal AI. |
-| [Chatbot using Next.js, OpenAI and Literal AI](/typescript/nextjs-openai/) | Observablity | Create a personalized and monitored chatbot with OpenAI, Next.js and Literal AI. |
-| [Chatbot using Vercel ai SDK and Literal AI](/typescript/vercel-ai-sdk/) | Observablity | Create a personalized and monitored chatbot with Vercel ai SDK and Literal AI. |
-| [Simple RAG using LanceDB, OpenAI and Literal AI](/typescript/lancedb-rag) | Observability | Create Simple RAG on Youtube Transcripts stored using LanceDB
\ No newline at end of file
+
+| Name                                                                                                    | Category      | Description                                                                                                |
+| ------------------------------------------------------------------------------------------------------- | ------------- | ---------------------------------------------------------------------------------------------------------- |
+| [Prompt Iteration with Promptfoo](/typescript/prompt-iteration-promptfoo/)                              | Evaluation    | Run a simple chat application and evaluate its results with two different prompt templates with Promptfoo. |
+| [Image Generation with Dall-E & Literal AI Attachment](/typescript/dall-e-image-generation)             | Observablity  | Use OpenAI Dall-E to Generate an image with a prompt and then send it to Literal AI.                       |
+| [Chatbot using Next.js, OpenAI and Literal AI](/typescript/nextjs-openai/)                              | Observablity  | Create a personalized and monitored chatbot with OpenAI, Next.js and Literal AI.                           |
+| [Chatbot using Vercel ai SDK and Literal AI](/typescript/vercel-ai-sdk/)                                | Observablity  | Create a personalized and monitored chatbot with Vercel ai SDK and Literal AI.                             |
+| [Simple RAG using LanceDB, OpenAI and Literal AI](/typescript/lancedb-rag)                              | Observability | Create Simple RAG on Youtube Transcripts stored using LanceDB                                              |
+| [Transcribing and summarizing audio with OpenAI Whisper, GPT-4 and Literal AI](/typescript/lancedb-rag) | Observability | Create a simple web app that transcribes and summarizes audio using OpenAI Whisper, GPT-4 and Literal AI.  |
```

Comment:
I like this description, very explicit.

However, it misses the part about emoji's. Not sure if it's important to mention the emoji part of make the title the same as the title of `typescript/audio-summarizer/README.md`

Reply:
Cool, love the new title. Feel free to resolve comment threads on Github as you see fit.